src/common/cuda_utils.h

```diff
@@ -11,6 +11,23 @@
 
 #if MXNET_USE_CUDA
 
+/*! \brief Macros/inlines to assist CLion to parse Cuda files (*.cu, *.cuh) */
+#ifdef __JETBRAINS_IDE__
```

Comment:
This allows CLion IDE to parse cuda code.  It has no net effect on anything else.